docs/source/gbd2017_models/causes/neoplasms/colon_and_rectum_cancer/cancer_model.rst

```diff
@@ -1,244 +1,274 @@
+   * - R
+     - Recovered
+     - Recovered from clinical colon and rectum cancer, not susceptible
+
+.. list-table:: State Data
+   :widths: 5 5 20 20
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - (1 - prev_PC - prev_C)
+     - 
+   * - S
+     - excess mortality rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     - 
+   * - PC
+     - prevalence
+     - prev_c441 * MST / AST
+     - 
+   * - PC
+     - excess mortality rate
+     - 0
+     - 
+   * - PC
+     - disabilty weights
+     - 0
+     - 
+   * - C
+     - prevalence
+     - prev_c441
+     - 
+   * - C
+     - excess mortality rate
+     - :math:`\frac{\text{csmr_c441}}{\text{prev_c441}}`
+     - 
+   * - C
+     - disabilty weights
+     - :math:`\frac{\displaystyle{\sum_{s\in\text{s_c441}}}\scriptstyle{\text{disability_weight}_s\,\times\,\text{prev}_s}}{\displaystyle{\sum_{s\in\text{s_c441}}}\scriptstyle{\text{prev}_s}}`
+     - weighted average of colon and rectum cancer disability weight over all sequelae including ids s_296, s_298, s_299, s_5519, s_5522, s_5525
+   * - R
+     - prevalence
+     - 0
+     - No initialization in R state
+   * - R
+     - excess mortality rate
+     - 0
+     - 
+   * - R
+     - disabilty weights
+     - 0
+     - 
+
+.. list-table:: Transition Data
+   :widths: 5 5 5 20 20
+   :header-rows: 1
+
+   * - Transition
+     - Source state
+     - Sink state
+     - Value
+     - Notes
+   * - i_pc
+     - S
+     - PC
+     - incidence_c441(age + MST)
+     - 
+   * - i_c
+     - PC
+     - C
+     - 1 / MST
+     - 
+   * - r
+     - C
+     - R
+     - ? per person-year for all ages and sexes
+     - 
+
+.. list-table:: Data sources
+   :widths: 5 20 20
+   :header-rows: 1
+   
+   * - Measure
+     - Sources
+     - Notes
+   * - prev_c441
+     - forecasted for future years 2020-2040
+     - forcasted data filepath: /ihme/costeffectiveness/vivarium_csu_cancer
+   * - incidence_c441
+     - forecasted for future years 2020-2040
+     - forcasted data filepath: /ihme/costeffectiveness/vivarium_csu_cancer
+   * - csmr_c441
+     - forecasted for future years 2020-2040
+     - forcasted data filepath: /ihme/costeffectiveness/vivarium_csu_cancer
+   * - remission_c441
+     - GBD 2017
+     - remission rate of cervical cancer = 1/? per person-years for all ages 
+     and sexes 
+   * - Disability weights for colon and rectum cancer
+     - GBD 2017 YLD appendix
+     - weighted average of colon and rectum cancer disability weight over all sequelae with ids s_296, s_298, s_299, s_5519, s_5522, s_5525
+   * - ACMR
+     - forecasted for future years 2020-2040 
+     - forcasted data filepath: /ihme/costeffectiveness/vivarium_csu_cancer
+   * - Population
+     - demography for 2017 
+     - mid-year population
+   * - MST
+     - 4.5-5.8 years (Brenner et al.)
+     - 
+   * - AST
+     - ?
+     - 
+
+Validation Criteria
++++++++++++++++++++
+

```

Comment:
I think we should add that incidence and prevalence of the clinical state should validate to GBD too

Reply:
Yes, as we always compare it to our GBD forecasted data.